extensions/wikia/ArticleVideo/styles/ooyala/components/_ad-screen.scss

```diff
@@ -0,0 +1,24 @@
+.oo-ad-screen {
```

Comment:
We send all stylesheets to the user, even the unused ones like this one. Is there a way to remove / disable them?